[PROPOSAL] test imports have some unmarked expected fails on win10 py3.7 pyme-depends 1.18, dev install

**Describe the bug**
<details> <summary> pytest output </summary>

```
(pyme) C:\code\python-microscopy\tests>pytest
================================================= test session starts =================================================
platform win32 -- Python 3.7.16, pytest-7.1.2, pluggy-1.0.0
rootdir: C:\code\python-microscopy
collected 122 items

PYME\test_imports.py F.F..                                                                                       [  4%]
PYME\Acquire\test_tweeter.py s                                                                                   [  4%]
PYME\Analysis\test_piezo_movement_correction.py ..                                                               [  6%]
PYME\Analysis\points\test_clumps.py .....                                                                        [ 10%]
PYME\Analysis\points\test_cluster_morphology.py ...                                                              [ 13%]
PYME\Analysis\points\test_coordinate_tools.py ..........                                                         [ 21%]
PYME\Analysis\points\test_multiview.py ..                                                                        [ 22%]
PYME\Analysis\points\test_ripleys.py ..                                                                          [ 24%]
PYME\Analysis\points\test_spherical_harmonics.py ...x...s                                                        [ 31%]
PYME\Analysis\points\test_traveling_salesperson.py .....                                                         [ 35%]
PYME\Analysis\points\DistHist\test_dist_hist.py ....                                                             [ 38%]
PYME\DSView\LUT\test_lut.py .                                                                                    [ 39%]
PYME\IO\test_cluster_spooler.py F                                                                                [ 40%]
PYME\IO\test_countdir.py F                                                                                       [ 40%]
PYME\IO\test_dataserver.py FFFFFFsE                                                                              [ 46%]
PYME\IO\test_h5.py ...F                                                                                          [ 50%]
PYME\IO\test_pipeline_io.py .X                                                                                   [ 51%]
PYME\IO\test_pzf.py .........                                                                                    [ 59%]
PYME\experimental\test_octree.py ...                                                                             [ 61%]
PYME\experimental\test_treap.py ..                                                                               [ 63%]
PYME\experimental\test_triangle_mesh.py sssssssssssssssss                                                        [ 77%]
PYME\experimental\test_triangular_mesh.py xxxxxxxxx                                                              [ 84%]
PYME\localization\test_MultifitSR.py .s                                                                          [ 86%]
PYME\localization\test_dumbell.py .                                                                              [ 86%]
PYME\localization\test_localization_algorithms.py ......                                                         [ 91%]
PYME\localization\FitFactories\Interpolators\test_interpolators.py ...                                           [ 94%]
PYME\recipes\test_acquisition.py .                                                                               [ 95%]
PYME\recipes\test_measurement.py ..X                                                                             [ 97%]
PYME\recipes\test_processing.py .                                                                                [ 98%]
PYME\recipes\test_recipes.py .                                                                                   [ 99%]
PYME\recipes\test_tablefilters.py .                                                                              [100%]

======================================================= ERRORS ========================================================
_________________________________ ERROR at teardown of test_mulithread_result_filing __________________________________

    def teardown_module():
        global proc, tmp_root
>       proc.send_signal(signal.SIGINT)

PYME\IO\test_dataserver.py:36:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <subprocess.Popen object at 0x0000020F933DEF48>, sig = <Signals.SIGINT: 2>

    def send_signal(self, sig):
        """Send a signal to the process."""
        # Don't signal a process that we know has already died.
        if self.returncode is not None:
            return
        if sig == signal.SIGTERM:
            self.terminate()
        elif sig == signal.CTRL_C_EVENT:
            os.kill(self.pid, signal.CTRL_C_EVENT)
        elif sig == signal.CTRL_BREAK_EVENT:
            os.kill(self.pid, signal.CTRL_BREAK_EVENT)
        else:
>           raise ValueError("Unsupported signal: {}".format(sig))
E           ValueError: Unsupported signal: 2

C:\Users\Administrator\anaconda3\envs\pyme\lib\subprocess.py:1335: ValueError
====================================================== FAILURES =======================================================
___________________________________________________ test_IO_imports ___________________________________________________

    @pytest.mark.xfail(not HAVE_WX, reason="Depends on wx, which is not installed on this platform")
    def test_IO_imports():
        from PYME.IO import buffer_helpers, buffers
        from PYME.IO import clusterExport, clusterGlob
        from PYME.IO import clusterIO, clusterListing, clusterResults, clusterUpload
        from PYME.IO import dataExporter, dataWrap, dcimg, h5rFile
        from PYME.IO import image, load_psf, MetaDataHandler, PZFFormat
>       from PYME.IO import ragged, tabular, unifiedIO, countdir

PYME\test_imports.py:20:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from PYME import config
    if config.get('cluster-listing-no-countdir', False):
        raise ImportError('Not using countdir')

>   from .countdir import *
E   ModuleNotFoundError: No module named 'PYME.IO.countdir.countdir'

..\PYME\IO\countdir\__init__.py:5: ModuleNotFoundError
_____________________________________________ test_DSView_modules_imports _____________________________________________

    @pytest.mark.xfail(not HAVE_WX, reason="Depends on wx, which is not installed on this platform")
    def test_DSView_modules_imports():
        from PYME.DSView import modules

        modLocations = {}
        for m in modules.localmodules:
            modLocations[m] = ['PYME', 'DSView', 'modules']

        #import all modules
        for modName, ml in modLocations.items():
>           __import__('.'.join(ml) + '.' + modName, fromlist=ml)

PYME\test_imports.py:42:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from ._base import Plugin
    import numpy as np

>   import neuroglancer
E   ModuleNotFoundError: No module named 'neuroglancer'


____________________________________________________ test_dirsize _____________________________________________________

    def test_dirsize():
>       from PYME.IO import countdir

PYME\IO\test_countdir.py:3:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from PYME import config
    if config.get('cluster-listing-no-countdir', False):
        raise ImportError('Not using countdir')

>   from .countdir import *
E   ModuleNotFoundError: No module named 'PYME.IO.countdir.countdir'

..\PYME\IO\countdir\__init__.py:5: ModuleNotFoundError

=============================================== short test summary info ===============================================
FAILED PYME\test_imports.py::test_IO_imports - ModuleNotFoundError: No module named 'PYME.IO.countdir.countdir'
FAILED PYME\test_imports.py::test_DSView_modules_imports - ModuleNotFoundError: No module named 'neuroglancer'
FAILED PYME\IO\test_cluster_spooler.py::test_spooler - AssertionError: No servers found for distribution. Make sure t...FAILED PYME\IO\test_countdir.py::test_dirsize - ModuleNotFoundError: No module named 'PYME.IO.countdir.countdir'
FAILED PYME\IO\test_dataserver.py::test_put - ValueError: attempt to get argmin of an empty sequence
FAILED PYME\IO\test_dataserver.py::test_putfiles_and_list - ValueError: attempt to get argmin of an empty sequence
FAILED PYME\IO\test_dataserver.py::test_list_after_timeout - ValueError: attempt to get argmin of an empty sequence
FAILED PYME\IO\test_dataserver.py::test_double_put - ValueError: attempt to get argmin of an empty sequence
FAILED PYME\IO\test_dataserver.py::test_aggregate_h5r - OSError: No data servers found, is the cluster running?
FAILED PYME\IO\test_dataserver.py::test_dircache_purge - ValueError: attempt to get argmin of an empty sequence
FAILED PYME\IO\test_h5.py::test_h5r - PermissionError: [WinError 32] The process cannot access the file because it is...ERROR PYME\IO\test_dataserver.py::test_mulithread_result_filing - ValueError: Unsupported signal: 2
============== 11 failed, 78 passed, 21 skipped, 10 xfailed, 2 xpassed, 1791 warnings, 1 error in 56.38s ==============

```
</details>


<details> <summary> conda list </summary>

```
(pyme) C:\Users\Administrator>conda list
# packages in environment at C:\Users\Administrator\anaconda3\envs\pyme:
#
# Name                    Version                   Build  Channel
alabaster                 0.7.12                   py37_0
atomicwrites              1.4.0                      py_0
attrs                     22.1.0           py37haa95532_0
babel                     2.11.0           py37haa95532_0
blas                      1.0                         mkl
blosc                     1.15.0               h7bd577a_0
bottleneck                1.3.5            py37h080aedc_0
brotli                    1.0.9                h2bbff1b_7
brotli-bin                1.0.9                h2bbff1b_7
brotlipy                  0.7.0           py37h2bbff1b_1003
bzip2                     1.0.6            vc14hdec8e7a_1  [vc14]  anaconda
ca-certificates           2023.01.10           haa95532_0
cairo                     1.14.12              hf171d8a_3
certifi                   2022.12.7        py37haa95532_0
cffi                      1.15.1           py37h2bbff1b_3
charset-normalizer        2.0.4              pyhd3eb1b0_0
cheroot                   8.5.2            py37haa95532_1
cherrypy                  18.6.0           py37haa95532_0
cloudpickle               2.0.0              pyhd3eb1b0_0
colorama                  0.4.6            py37haa95532_0
cryptography              38.0.4           py37h21b164f_0
cycler                    0.11.0             pyhd3eb1b0_0
cython                    0.29.32          py37hd77b12b_0
cytoolz                   0.12.0           py37h2bbff1b_0
dask-core                 2021.10.0          pyhd3eb1b0_0
django                    2.1                      py37_0
docutils                  0.18.1           py37haa95532_3
fftw                      3.3.4                         0    david_baddeley
flit-core                 3.6.0              pyhd3eb1b0_0
fonttools                 4.25.0             pyhd3eb1b0_0
freetype                  2.10.4               hd328e21_0
fsspec                    2022.11.0        py37haa95532_0
future                    0.18.2                   py37_1
hdf5                      1.8.18           vc14h7a021fe_0  [vc14]  anaconda
icc_rt                    2022.1.0             h6049295_2
icu                       58.2             vc14hc45fdbb_0  [vc14]  anaconda
idna                      3.4              py37haa95532_0
ifaddr                    0.1.6                      py_0    david_baddeley
imageio                   2.19.3           py37haa95532_0
imagesize                 1.4.1            py37haa95532_0
importlib-metadata        4.11.3           py37haa95532_0
importlib_metadata        4.11.3               hd3eb1b0_0
iniconfig                 1.1.1              pyhd3eb1b0_0
intel-openmp              2021.4.0          haa95532_3556
jaraco.classes            3.2.1              pyhd3eb1b0_0
jaraco.collections        3.3.0              pyhd3eb1b0_0
jaraco.functools          3.3.0              pyhd3eb1b0_0
jaraco.text               3.5.1            py37haa95532_2
jinja2                    3.1.2            py37haa95532_0
joblib                    1.1.1            py37haa95532_0
jpeg                      9b               vc14h4d7706e_1  [vc14]  anaconda
kiwisolver                1.4.4            py37hd77b12b_0
libbrotlicommon           1.0.9                h2bbff1b_7
libbrotlidec              1.0.9                h2bbff1b_7
libbrotlienc              1.0.9                h2bbff1b_7
libpng                    1.6.37               h2a8f88b_0
libtiff                   4.0.8           vc14h04e2a1e_10  [vc14]  anaconda
libwebp                   1.2.4                h2bbff1b_0
libwebp-base              1.2.4                h2bbff1b_0
locket                    1.0.0            py37haa95532_0
lzo                       2.10             vc14h0a64fa6_1  [vc14]  anaconda
markupsafe                2.1.1            py37h2bbff1b_0
matplotlib                3.5.2            py37haa95532_0
matplotlib-base           3.5.2            py37hd77b12b_0
mkl                       2021.4.0           haa95532_640
mkl-service               2.4.0            py37h2bbff1b_0
mkl_fft                   1.3.1            py37h277e83a_0
mkl_random                1.2.2            py37hf11a4ad_0
more-itertools            8.12.0             pyhd3eb1b0_0
mpld3                     v0.3                     py37_0    david_baddeley
munkres                   1.1.4                      py_0
networkx                  2.6.3              pyhd3eb1b0_0
numexpr                   2.8.4            py37h5b0cc5e_0
numpy                     1.21.5           py37h7a0a035_3
numpy-base                1.21.5           py37hca35cd5_3
openssl                   1.1.1s               h2bbff1b_0
packaging                 22.0             py37haa95532_0
pandas                    1.3.5            py37h6214cd6_0
partd                     1.2.0              pyhd3eb1b0_1
pillow                    9.3.0            py37hdc2b20a_1
pip                       22.3.1           py37haa95532_0
pixman                    0.34.0           vc14h00fde18_1  [vc14]  anaconda
pluggy                    1.0.0            py37haa95532_1
portend                   2.7.1              pyhd3eb1b0_0
psutil                    5.9.0            py37h2bbff1b_0
py                        1.11.0             pyhd3eb1b0_0
pycairo                   1.19.1           py37hb2538c6_0
pycparser                 2.21               pyhd3eb1b0_0
pyface                    7.1.0                    py37_1
pyfftw                    0.11.1+26.g101c792          pypi_0    pypi
pygments                  2.11.2             pyhd3eb1b0_0
pyme-depends              1.18                     py37_0    david_baddeley
pymecompress              0.3.5                    py37_0    david_baddeley
pyopengl                  3.1.1a1          py37haa95532_0
pyopenssl                 22.0.0             pyhd3eb1b0_0
pyparsing                 3.0.9            py37haa95532_0
pyqt                      5.9.2            py37hd77b12b_6
pysocks                   1.7.1                    py37_1
pytables                  3.4.4            py37h0f9c03a_0
pytest                    7.1.2            py37haa95532_0
python                    3.7.16               h6244533_0
python-dateutil           2.8.2              pyhd3eb1b0_0
python-microscopy         22.10.17.post0.dev0           dev_0    <develop>
pytz                      2022.7           py37haa95532_0
pywavelets                1.3.0            py37h2bbff1b_0
pywin32                   305              py37h2bbff1b_0
pyyaml                    5.3.1            py37he774522_0
qt                        5.9.7            vc14h73c81de_0  [vc14]  anaconda
repoze.lru                0.7              py37haa95532_0
requests                  2.28.1           py37haa95532_0
routes                    2.5.1            py37haa95532_0
scikit-image              0.16.2           py37h47e9c7a_0
scikit-learn              1.0.2            py37hf11a4ad_1
scipy                     1.7.3            py37h0a974cb_0
setuptools                65.6.3           py37haa95532_0
simplejson                3.17.6           py37h2bbff1b_0
sip                       4.19.13          py37hd77b12b_0
six                       1.16.0             pyhd3eb1b0_1
snappy                    1.1.7            vc14h2dea872_1  [vc14]  anaconda
snowballstemmer           2.2.0              pyhd3eb1b0_0
sphinx                    5.0.2            py37haa95532_0
sphinxcontrib-applehelp   1.0.2              pyhd3eb1b0_0
sphinxcontrib-devhelp     1.0.2              pyhd3eb1b0_0
sphinxcontrib-htmlhelp    2.0.0              pyhd3eb1b0_0
sphinxcontrib-jsmath      1.0.1              pyhd3eb1b0_0
sphinxcontrib-qthelp      1.0.3              pyhd3eb1b0_0
sphinxcontrib-serializinghtml 1.1.5              pyhd3eb1b0_0
sqlite                    3.40.1               h2bbff1b_0
tempora                   4.1.1              pyhd3eb1b0_0
threadpoolctl             2.2.0              pyh0d69192_0
tk                        8.6.7            vc14hb68737d_1  [vc14]  anaconda
tomli                     2.0.1            py37haa95532_0
toolz                     0.12.0           py37haa95532_0
toposort                  1.5                      py37_3    david_baddeley
tornado                   6.2              py37h2bbff1b_0
traits                    6.2.0            py37h2bbff1b_0
traitsui                  7.1.0                      py_0
typing_extensions         4.4.0            py37haa95532_0
ujson                     5.4.0            py37hd77b12b_0
urllib3                   1.26.14          py37haa95532_0
vc                        14.2                 h21ff451_1
vs2015_runtime            14.27.29016          h5e58377_2
wheel                     0.37.1             pyhd3eb1b0_0
win_inet_pton             1.1.0            py37haa95532_0
wincertstore              0.2              py37haa95532_2
wxpython                  4.0.4            py37ha925a31_0
yaml                      0.1.7            vc14h4cb57cf_1  [vc14]  anaconda
zc.lockfile               2.0                pyhd3eb1b0_0
zeroconf                  0.24.0                   py37_3    david_baddeley
zipp                      3.11.0           py37haa95532_0
zlib                      1.2.11           vc14h1cdd9ab_1  [vc14]  anaconda
```
</details>

**To Reproduce**
Steps to reproduce the behavior:
1. `conda create -n pyme python=3.7 pyme-depends`
2. install msvc business
3. clone the head
4. `python setup.py develop`
5. `conda install pytest`
6. some failures

**Expected behavior**
pass the tests.

Less worried about cluster tests given I haven't double checked firewall issues yet, etc., but would ideally not hit import errors on countdir or neuroglancer.

**proposal**
- Unless we want to ship neuroglancer with pyme-depends we should mark the neuroglancer dsview module import test as skip if neuroglancer can't be imported
- unless we want to write a `countdir.c` for windows we should check `sys.win32`, separate countdir import test from `test_IO_imports`

--- a/PYME/Acquire/ui/HDFSpoolFrame.py
+++ b/PYME/Acquire/ui/HDFSpoolFrame.py
@@ -226,7 +226,7 @@ class PanSpool(afp.foldingPane):
     
         self.bStartSpool = wx.Button(pan, -1, 'Start', style=wx.BU_EXACTFIT)
         self.bStartSpool.Bind(wx.EVT_BUTTON, self.OnBStartSpoolButton)
-        self.bStartSpool.SetDefault()
+        #self.bStartSpool.SetDefault()
         hsizer.Add(self.bStartSpool, 0, wx.ALL | wx.ALIGN_CENTER_VERTICAL, 2)
     
         # self.bStartStack = wx.Button(pan,-1,'Z-Series',style=wx.BU_EXACTFIT)

--- a/PYME/DSView/modules/neuroglancer.py
+++ b/PYME/DSView/modules/neuroglancer.py
@@ -1,8 +1,15 @@
 from ._base import Plugin
 import numpy as np
+import logging
+logger = logging.getLogger(__name__)
 
-import neuroglancer
-from neuroglancer import local_volume
+try:
+    import neuroglancer
+    from neuroglancer import local_volume
+except ImportError:
+    # this is to keep tests which automatically import all modules happy.
+    logger.exception('Could not import neuroglancer, install if you need PYMEImage neuroglancer connectivity')
+    # NB - will cause a subsequent error when the code tries to access the modules
 
 
 class PyramidLocalVolume(local_volume.LocalVolume):

